install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -175,3 +174,5 @@ DELETE FROM `PREFIX_configuration` WHERE `name` = 'PS_CACHEFS_DIRECTORY_DEPTH';
 DELETE FROM `PREFIX_configuration` WHERE `name` = 'PS_CART_REDIRECT';
 
 INSERT INTO `PREFIX_hook` (`name`, `title`, `description`, `position`) VALUES ('validateCustomerFormFields', 'Customer registration form validation', 'This hook is called to a module when it has sent addtionnal fields with additionalCustomerFormFields', '1');
+/* PHP:remove_cipher_algorithm(); */;
```

Comment:
Ahh, I can recall changing an other PR because of that. I should get rid of this habit :stuck_out_tongue_winking_eye: